obs_status.yml

```diff
@@ -0,0 +1,1995 @@
+  mp_starcat_time: 2005:056:20:11:01.041
+  status: 0
+- agasc_id:
+  - 723657296
+  comments: Faint star. Almost never tracked. Set mag to 11.547
+  mp_starcat_time: 2005:074:22:44:33.953
+  status: 1
+- agasc_id:
+  - 652480464
+  comments: Never acquired. Set mag to MAXMAG
+  mp_starcat_time: 2005:150:01:28:35.423
+  status: 1
+- agasc_id:
+  - 615914552
+  comments: Never acquired. Set mag to MAXMAG
+  mp_starcat_time: 2005:281:04:44:20.882
+  status: 1
+- agasc_id:
+  - 839385704
+  comments: Almost never acquired. When acquired, it was too bright. Star has been
+    observed other times. Known symbiotic star
+  mp_starcat_time: 2005:282:18:53:05.394
+  status: 1
+- agasc_id:
+  - 1197635184
+  comments: Never acquired. Set mag to MAXMAG
+  mp_starcat_time: 2005:353:06:48:26.153
+  status: 1
+- agasc_id:
+  - 1197635184
+  comments: Never acquired. Set mag to MAXMAG
+  mp_starcat_time: 2005:354:14:42:59.384
+  status: 1
+- agasc_id:
+  - 1197635184
+  comments: Never acquired. Set mag to MAXMAG
+  mp_starcat_time: 2005:355:23:10:36.631
+  status: 1
+- agasc_id:
+  - 118498608
+  comments: Faint star. Affected by imposters. Set mag to MAXMAG
+  mp_starcat_time: 2006:014:13:25:39.795
+  status: 1
+- agasc_id:
+  - 330967424
+  comments: Never acquired. Set mag to MAXMAG
+  mp_starcat_time: 2006:064:07:51:01.152
+  status: 1
+- agasc_id:
+  - 46794712
+  comments: Faint star affected by imposter. Marginally acceptable
+  mp_starcat_time: 2006:159:12:19:26.972
+  status: 0
+- agasc_id:
+  - 46794712
+  comments: Faint star affected by imposter. Marginally acceptable
+  mp_starcat_time: 2006:159:22:22:16.972
+  status: 0
+- agasc_id:
+  - 48892720
+  comments: Faint star. Affected by imposter. Set mag to MAXMAG
+  mp_starcat_time: 2006:162:12:54:27.115
+  status: 1
+- agasc_id:
+  - 763893760
+  comments: Mostly tracking 11.6 mag star; supplement estimate is 13.8 mag
+  mp_starcat_time: 2006:212:07:39:07.107
+  status: 1
+- agasc_id:
+  - 1117787424
+  comments: Never acquired. Set mag to MAXMAG
+  mp_starcat_time: 2006:220:05:35:23.362
+  status: 1
+- agasc_id:
+  - 1117787424
+  comments: Never acquired. Set mag to MAXMAG
+  mp_starcat_time: 2006:220:06:30:11.687
+  status: 1
+- agasc_id:
+  - 1117787424
+  comments: Never acquired. Set mag to MAXMAG
+  mp_starcat_time: 2006:220:07:13:11.687
+  status: 1
+- agasc_id:
+  - 1117787424
+  comments: Never acquired. Set mag to MAXMAG
+  mp_starcat_time: 2006:220:07:56:11.687
+  status: 1
+- agasc_id:
+  - 1117787424
+  comments: Never acquired. Set mag to MAXMAG
+  mp_starcat_time: 2006:220:08:39:11.687
+  status: 1
+- agasc_id:
+  - 1117787424
+  comments: Never acquired. Set mag to MAXMAG
+  mp_starcat_time: 2006:220:09:22:11.687
+  status: 1
+- agasc_id:
+  - 879768704
+  comments: Almost never tracked. Set mag to MAXMAG
+  mp_starcat_time: 2007:085:18:25:58.626
+  status: 1
+- agasc_id:
+  - 1020796600
+  comments: Never acquired; set mag 11.5 +/- 0.1
```

Comment:
11.5 is basically MAXMAG

Reply:
I think so. This is one of the early ones Tom did before he suggested the MAXMAG solution, and because it does not have the keyword it fell through the cracks.